homebrew_challenge/src/components/Footer/Footer.jsx

```diff
@@ -131,8 +132,12 @@ const Footer = ({ setCurrentPage }) => {
               </a>
             </div>
           </Col>
-          <Col sm={12} md={3} className="d-flex justify-content-end social">
-            <div className="d-flex align-items-center">Connect with us:</div>
+          <Col
+            sm={12}
+            md={3}
+            className="d-flex justify-content-end social align-items-center"
+          >
+            <div className="d-flex">Connect with us:</div>
```

Comment:
the d-flex isn't needed here - there's only one string in the div

Reply:
✅